WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift

```diff
@@ -1,11 +1,45 @@
 import UIKit
+import WordPressShared
 
 class LoginEpilogueViewController: UIViewController {
     var originalPresentingVC: UIViewController?
     var dismissBlock: ((_ cancelled: Bool) -> Void)?
+    @IBOutlet var buttonPanel: UIView?
+    @IBOutlet var shadowView: UIView?
 
     // @IBAction to allow to set the selector for target in the storyboard
     @IBAction func unwindOut(segue: UIStoryboardSegue) {
         dismissBlock?(false)
     }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        colorPanelBasedOnTableViewContents()
+    }
+
+    func colorPanelBasedOnTableViewContents() {
+        var tableVC: UITableViewController?
+        for childVC in childViewControllers {
+            if let childVC = childVC as? UITableViewController {
+                tableVC = childVC
+            }
+        }
+
+        guard let table = tableVC?.tableView,
+            let buttonPanel = buttonPanel else {
+            return
+        }
+
+        let contentSize = table.contentSize
+        let screenHeight = UIScreen.main.bounds.size.height
+        let panelHeight = buttonPanel.frame.size.height
+
+        if screenHeight - panelHeight > contentSize.height {
+            buttonPanel.backgroundColor = WPStyleGuide.lightGrey()
+            shadowView?.isHidden = true
+        } else {
+            buttonPanel.backgroundColor = UIColor.white
+            shadowView?.isHidden = false
+        }
+    }
 }
```

Comment:
When this VC is dismissed the initial username and password screen is visible for a few seconds, again maybe something you are planning to address on a future PR, but just in case I report it.
![loginscreenreappearing](https://cloud.githubusercontent.com/assets/5558824/25965674/0e988a46-365f-11e7-964d-ba00fdebb17b.gif)